crates/query-engine/metadata/src/metadata/database.rs

```diff
@@ -9,14 +9,13 @@ pub struct CompositeTypes(pub BTreeMap<String, CompositeType>);
 
 /// A Scalar Type.
 #[derive(Debug, Clone, PartialEq, Eq, PartialOrd, Ord)]
-
-pub struct ScalarType(pub String);
+pub struct ScalarTypeName(pub String);
```

Comment:
In order to be precise in our terminology, we now take care to point out that we are referring explicitly to the name of a scalar type.

This distinction will become more relevant in subsequent PRs when we distinguish between the scalar type name at the ndc level vs on the database.